src/neuroconv/datainterfaces/ecephys/edf/edfdatainterface.py

```diff
@@ -23,7 +25,22 @@ def get_source_schema(cls) -> dict:
         source_schema["properties"]["file_path"]["description"] = "Path to the .edf file."
         return source_schema
 
-    def __init__(self, file_path: FilePath, verbose: bool = True, es_key: str = "ElectricalSeries"):
+    def _source_data_to_extractor_kwargs(self, source_data: dict) -> dict:
+
+        extractor_kwargs = source_data.copy()
+        extractor_kwargs.pop("channels_to_skip")
+        extractor_kwargs["all_annotations"] = True
+        extractor_kwargs["use_names_as_ids"] = True
```

Comment:
@bendichter 
This is necessary to display meaningful names that the user can suppress with the newly added parameter. Otherwise, the channel ids are "1", "2", "3", etc.

This is technically a breaking change because we are changing the way we add the data to the NWBFile but I also consider this an improvement:

![image](https://github.com/user-attachments/assets/260005a2-85ad-44c5-b19a-5c121423996a)